rl_policy:
  ros__parameters:
    actor_path: 'overtaking_models/baseline_new/123/models/final/TD3_actor.pht'
    critic_path: 'overtaking_models/baseline_new/123/models/final/TD3_critic.pht'
    algorithm: 'TD3'
    observation_mode: 'lidar_only'
